CHANGELOG.md

```diff
@@ -4,6 +4,494 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [0.16.0] - 2022-08-22
+
+## ❗ BREAKING ❗
+
+### Remove QueryPlannerService ([PR #1552](https://github.com/apollographql/router/pull/1552))
+
+This service was redundant, since anything done as part of the `QueryPlannerService` could be done either at the `SupergraphService` or at the `ExecutionService` level.
+
+By [@o0Ignition0o](https://github.com/o0Ignition0o)
```

Comment:
Sometimes we have pattern like `By [...](...) in URL_OF_THE_PR` what's the good way ?

Reply:
I thought it was there to show the list of contributors in the release page, but the last time it worked was for 0.14?
@abernix any context on that?

Reply:
I suggested the `in URL_OF_THE_PR` on a lot of the requests below.  Unfortunately, to make the contributors list work, this format has to be edited manually searched and replaced be _real_ `@mentions` when copying and pasting the contents into the GitHub release.  So yet another thing to automate.

(The other side of the coin is that, frustratingly, bare `@mention`s don't work in the `CHANGELOG.md` itself, which is why the link format is nice so it's clickable — and to keep track of the authors. 😭 )